examples/nginx-ingress-scrape-config.yaml

```diff
@@ -0,0 +1,53 @@
+- job_name: 'nginx-ingress'
+  kubernetes_sd_configs:
+    - role: pod
+  relabel_configs:
+    - source_labels: [__meta_kubernetes_pod_label_nsm_nginx_com_enable_ingress]
+      action: keep
+      regex: true
+    - source_labels: [__meta_kubernetes_pod_label_nsm_nginx_com_enable_egress]
+      action: keep
+      regex: true
+    - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
+      action: keep
+      regex: true
+    - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
+      action: replace
+      target_label: __address__
+      regex: (.+)(?::\d+);(\d+)
+      replacement: $1:$2
+    - source_labels: [__meta_kubernetes_namespace]
+      action: replace
+      target_label: namespace
+    - source_labels: [__meta_kubernetes_pod_name]
+      action: replace
+      target_label: pod
+    - action: labelmap
+      regex: __meta_kubernetes_pod_label_nsm_nginx_com_(.+)
+    - action: labeldrop
+      regex: __meta_kubernetes_pod_label_nsm_nginx_com_(.+)
+    - action: labelmap
+      regex: __meta_kubernetes_pod_label_(.+)
+    - action: labelmap
+      regex: __meta_kubernetes_pod_annotation_nsm_nginx_com_enable_(.+)
+  metric_relabel_configs:
+    - source_labels: [__name__]
+      regex: 'nginx_ingress_controller_upstream_server_response_latency_ms(.+)'
+      target_label: __name__
+      replacement: 'nginxplus_upstream_server_response_latency_ms$1'
+    - source_labels: [__name__]
+      regex: 'nginx_ingress_nginxplus(.+)'
```

Comment:
Few references of nginxplus in here...we need to verify which metrics are actually available to OSS. This scrape config is likely different from the plus version.

Reply:
Should look better now